optimizer_scripts/tools/common_pattern.py

```diff
@@ -116,6 +116,8 @@ def make_UpsamplingBilinear2d_value_info(g, resize_node_name):
     resize_node = helper.find_node_by_node_name(g, resize_node_name)
 
     shape_data_node = helper.find_node_by_output_name(g, resize_node.input[3])
+    if (shape_data_node==None or shape_data_node.op_type!="Constant"):
+        return
```

Comment:
Please use python style coding. 
if shape_data_node is None or shape_data_node.op_type != "Constant":